src/pages/index.js

```diff
@@ -26,29 +28,63 @@ const profilePopup = new PopupWithForm(
 profilePopup.setEventListeners();
 const newCardPopup = new PopupWithForm("#newcards", addNewCardSubmit);
 newCardPopup.setEventListeners();
+const editPfp = new PopupWithForm("#pfpEdit", editPfpSubmit);
+editPfp.setEventListeners();
+const confirmPopup = new PopupWithConfirm("#confirm");
+confirmPopup.setEventListeners();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "8d3dab07-3d12-4401-9299-225a306c1793",
+    "Content-Type": "application/json",
+  },
+});
+const userInfo = new UserInfo(profileName, profileDescription, pfp);
 
-function handleCardImageClick(name, link) {
-  imagePopup.open(name, link);
+//initalization
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#jscard-template",
+    handleCardImageClick,
+    handleDeleteCard,
+    handleLikeCard
+  );
+  const cardElement = card.getView();
+  cardSection.addItem(cardElement);
+}
+//render newcards
+function cardListData() {
+  return api
+    .getInitialCards()
+    .then((data) => {
+      const cardElement = data.map((cardData) => createCard(cardData));
+      cardSection.renderItems(cardElement);
```

Comment:
* You haven't corrected my remark https://github.com/omegax117/se_project_aroundtheus/pull/1#discussion_r1671744255  You need to use `renderItems` for rendering the array of cards. 


It should be just

```js
    .then((data) => {
      cardSection.renderItems(data);
    })

```